api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -84,6 +99,29 @@
     (is (not (:started workload)))
     (is (:started (workloads/start-workload! workload)))))
 
+;; Flag for team discussion
+;; org.postgresql.util.PSQLException: ERROR:
+;; null value in column "id" of relation "terradatareposourcedetails_000000001" violates not-null constraint
+(deftest test-update-tdr-source
+  (let [{:keys [source]}
+        (workloads/create-workload!
+         (workloads/covid-workload-request {} {} {}))]
+    (with-redefs-fn
+      {#'covid/create-snapshots mock-create-snapshots
+       #'covid/find-new-rows mock-find-new-rows
+       #'covid/check-tdr-job mock-check-tdr-job}
+      #(#'covid/update-tdr-source source))
```

Comment:
Can you cal multi-method instead please? I want to make sure it's plumbed in correctly!

Reply:
Ah sure, will update!